### PR TITLE
Update to version 1.4.4

### PR DESCRIPTION
**Pull request by gillins**
And use entry points for command line utilities.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,11 +7,15 @@ package:
 source:
   fn: rios-{{ version }}.tar.gz
   url: https://bitbucket.org/chchrsc/rios/downloads/rios-{{ version }}.tar.gz
-  md5: c6d370262a2630ef81deb53b88e5d184
+  sha256: 59fc71a3cfdc2c6de47fd38bc999bdb6de48ce882c5a680dac97118bf86b486f
 
 build:
-  number: 4
+  number: 0
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Do you want to try `noarch: python` here? Seems like `rios` is a good candidate for it.

Comment (gillins) [reply recipe/meta.yaml]: Indeed! Good thought

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,11 +7,15 @@ package:
...
   fn: rios-{{ version }}.tar.gz
   url: https://bitbucket.org/chchrsc/rios/downloads/rios-{{ version }}.tar.gz
-  md5: c6d370262a2630ef81deb53b88e5d184
+  sha256: 59fc71a3cfdc2c6de47fd38bc999bdb6de48ce882c5a680dac97118bf86b486f
 
 build:
-  number: 4
-  script: python setup.py install
+  number: 0
+  noarch: python
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You'll need to re-render to get the CI configure to use only CircleCI.

Comment (gillins) [reply recipe/meta.yaml]: Actually how does that work with entry points? We have a couple of command line programs now that use entry points...

Comment (gillins) [reply recipe/meta.yaml]: The Windows version will need .exe's created to run the entry points.

Comment (ocefpaf) [reply recipe/meta.yaml]: Int theory latest `conda` deals with that. I haven't test it thoroughly though.

**Comment by gillins** [comment]
`noarch` and entry points don't seem to work even on Linux... Gone back to original approach for now. Any thoughts?

**Comment by ocefpaf** [comment]
> `noarch` and entry points don't seem to work even on Linux... Gone back to original approach for now. Any thoughts?

It should but we clearly need more testing. See https://gitter.im/conda-forge/conda-forge.github.io?at=59afea188f4427b462dacc94 for my experience so far.

**Comment by gillins** [comment]
OK that is weird. Maybe something about this recipe makes it fail?

**Comment by gillins** [comment]
Ah ok I think it is because we are getting both entry points created and scripts installed. $RIOS_NOCMDLINE needs to be set to prevent the scripts getting installed (so they don't conflict with entry points on Unix). Hence the setting of this in `build.sh`/`bld.bat`. Maybe we need a better design in RIOS?

**Comment by ocefpaf** [comment]
> Edit: I'm assuming `build.sh`/`bld.bat` isn't run with `noarch: python` and `python setup.py install` run instead...

`bld.bat` but for but I think that without the `script` section in the recipe `conda-build` still runs `build.sh`.

> Maybe we need a better design in RIOS?

I'll take a better look tomorrow. Time to :sleeping: here.

**Comment by gillins** [comment]
> conda-build still runs build.sh.

I just checked and you are of course 100% correct. Mysterious...

**Comment by gillins** [comment]
I just had more of a look and it seems to me the entry points are not created with `noarch: python` and I can't work out why. Merging for now - perhaps we can work on this in a separate PR?

**Comment by ocefpaf** [comment]
Did you search for the entrypoints in the tarball conda package or did you install and test it?

I believe that latest conda create them at install time for `noarch` packages.

**Comment by gillins** [comment]
It doesn't seem to get as far as building a tarball. I've created a new PR to test at https://github.com/conda-forge/rios-feedstock/pull/13

**Pull request merged**